.travis.yml

```diff
@@ -64,13 +64,13 @@ jobs:
         - set -o errexit
         - BUILD_DIR=${BUILD_DIR:-.}
         - mkdir -p ${BUILD_DIR} && cd ${BUILD_DIR}
-        - cmake ${TRAVIS_BUILD_DIR}
+        - cmake ${TRAVIS_BUILD_DIR} -G Ninja
             -DCMAKE_C_FLAGS="$COMPILER_FLAGS"
             -DCMAKE_CXX_FLAGS="$COMPILER_FLAGS"
             -DCMAKE_EXE_LINKER_FLAGS="$LINKER_FLAGS"
             -DCMAKE_INSTALL_PREFIX=install
             -DCMAKE_EXPORT_COMPILE_COMMANDS=ON
-        - make -j2 install
+        - ninja install
```

Comment:
what's the default number of cores used on this system?

(when I run locally ninja says it would use 10 cores on my laptop, which is reasonable, so the default might be reasonable here too, but would be good to check)

Reply:
I know know exactly how many core the tavis machines have but one of the nice things about this change is that ninja will use all of them, regardless of how many there are.  So its more flexible that hardcoding "2".